smcontroller/smcontroller.go

```diff
@@ -340,6 +314,89 @@ func (controller *Controller) RegisterSM(stream pb.SMService_RegisterSMServer) e
  * Private
  **********************************************************************************************************************/
 
+func (controller *Controller) getSystemLog(logRequest cloudprotocol.RequestLog) error {
+	if logRequest.Filters.NodeIDs != nil {
+		for _, nodeID := range *logRequest.Filters.NodeIDs {
```

Comment:
maybe create a slice of nodes that will be used to get log and then call getSystemLog.
in this case, we will have only one call getSystemLog in the source code.